content/docs/command-reference/exp/branch.md

```diff
@@ -15,29 +15,29 @@ positional arguments:
 
 ## Description
 
-Makes a
-[Git branch](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging)
-off the last commit (`HEAD`) based on the given `experiment`, using the `branch`
-name provided. This makes the given experiment
-[persistent](/doc/user-guide/experiment-management#persistent-experiments) in
-the repo.
-
-In most cases this is similar to using `dvc exp apply` (applies the `experiment`
-results to the workspace) followed by Git branching and committing, except that
-`dvc exp branch` **does not** switch into the created `branch`. Equivalent to
-this:
-
-```bash
-[master] $ git checkout -b <branch>
-[branch] $ dvc exp apply <experiment>
-[branch] $ git add . && git commit
-[branch] $ git checkout master
-```
-
-For [checkpoints](/doc/command-reference/exp/run#checkpoints), the `experiment`
-(custom Git branch with multiple commits) is merged into the new `branch`.
-
-To switch into the new branch, use `git checkout branch` and `dvc checkout`.
+Makes a named Git
+[`branch`](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging)
+containing the target `experiment` (making it
+[persistent](/doc/user-guide/experiment-management#persistent-experiments)). For
+[checkpoint experiments](/doc/command-reference/exp/run#checkpoints), the new
+branch will contain multiple commits (the checkpoints).
+
+The new `branch` will be based on the experiment's parent commit (`HEAD` at the
+time that the experiment was run). Note that unlike `git branch`, DVC **does
+not** switch into the new `branch`.
```

Comment:
```suggestion
time that the experiment was run).
```
`git branch` also doesn't switch to the new branch, right? Is the point of this paragraph to clarify that the current HEAD/branch has no relation to `exp branch`?

Reply:
Ah yeah I was thinking `git checkout -b` I guess. Fixed

The paragraph clarifies where the branch is created, yes. Since the fact that exps are already commits is hidden from the user, it shouldn't be obvious to them that this just relabels an existing custom Git ref. or what it's tree is.